src/hotspot/share/asm/register.hpp

```diff
@@ -82,7 +82,7 @@ const type name = ((type)value)
 // For definitions of RegisterImpl* instances. To be redefined in an
 // OS-specific way.
 #ifdef __GNUC__
-#define INTERNAL_VISIBILITY  __attribute__ ((visibility ("internal")))
+#define INTERNAL_VISIBILITY  [[gnu::visibility ("internal")]]
```

Comment:
Don't make this change.  All this does is force uses of the macro to be at the beginning rather than the
end of the declarations where it is being applied.  Not worth the code churn.